base/floatfuncs.jl

```diff
@@ -170,10 +170,11 @@ end
 
 # isapprox: approximate equality of numbers
 """
-    isapprox(x, y; rtol::Real=sqrt(eps), atol::Real=0)
+    isapprox(x, y; rtol::Real=sqrt(eps), atol::Real=0, nans::Bool=false)
 
 Inexact equality comparison: `true` if `norm(x-y) <= atol + rtol*max(norm(x), norm(y))`. The
-default `atol` is zero and the default `rtol` depends on the types of `x` and `y`.
+default `atol` is zero and the default `rtol` depends on the types of `x` and `y`. The default
+`nans` is false, meaning `isapprox(NaN,NaN) == false`; call with `nans=true` to have `isapprox(NaN,NaN) == true`.
```

Comment:
How about "Keyword argument `nans` (defaults to false) determines whether or not `NaN` values are considered equal to one another."  ?